spec/mailers/previews/event_mailer_preview.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class EventMailerPreview < ActionMailer::Preview
+  def monthly_donation_summary
+    EventMailer.with(event: Event.not_demo_mode.first).monthly_donation_summary
```

Comment:
```suggestion
    EventMailer.with(event: Donation.last.event).monthly_donation_summary
```
So there's a donation basically always?